firebase-perf/src/test/java/com/google/firebase/perf/transport/TransportManagerTest.java

```diff
@@ -1399,6 +1400,9 @@ private void initializeTransport(boolean shouldInitialize) {
     if (shouldInitialize) {
       // Set the version name since Firebase sessions needs it.
       Context context = ApplicationProvider.getApplicationContext();
+
+      context.getApplicationInfo().uid = Process.myUid();
```

Comment:
nit: Can you leave a comment on why setting the uid is needed? Does the uid make the process_name consistent in test runs?

Reply:
For unit test application info does not application info related to uid, so we have to force set it. Yup I'll update the comment

Reply:
Done